### PR TITLE
Feat/create issue

### PR DESCRIPTION
**Pull request by sklirg**
* Implements a super basic admin create issue page.
* Adds a new item in state to keep track of editing/creation efforts of new issue
* Adds event to send issue to backend

Connected to https://github.com/vaas-org/vaas-server/pull/11

### CODE REVIEW

File: src/Page/App.elm [hunk 1]
```diff
@@ -27,6 +38,20 @@ body model =
 
 view : Model -> Html Msg
 view model =
+    let
+        isAdmin =
+            case model.client of
+                Just client ->
+                    case client.username of
+                        Just username ->
+                            username == "admin"
```
Comment (duvholt) [inline src/Page/App.elm:47]: Hackproof

File: src/Page/App.elm [hunk 2]
```diff
@@ -27,6 +38,20 @@ body model =
 
 view : Model -> Html Msg
 view model =
+    let
+        isAdmin =
+            case model.client of
```
Comment (duvholt) [inline src/Page/App.elm:43]: I'd probably have written the check like this:
```elm
isAdmin =
  model.client
    |> Maybe.andThen (\c -> c.username)
    |> Maybe.map(\username -> username == "admin") -- or Maybe.map ((==) "admin")
    |> Maybe.withDefault False
```
Doesn't matter much in this case, but I'm just curious if you know what the best practice in Elm is.

**Review by duvholt — APPROVED** [review]
:shipit:

File: src/Page/App.elm (hunk 1 above)
Comment (sklirg) [reply src/Page/App.elm:47]: :shipit:

**Pull request merged**